README.md

```diff
@@ -21,12 +21,16 @@
 
 ## Usage
 
-Those 3 nugets provide a support for the [M5Stack](https://docs.m5stack.com/en/products?id=core), [M5StickC](https://docs.m5stack.com/en/core/m5stickc), [M5StickCPlus](https://docs.m5stack.com/en/core/m5stickc_plus) and [M5Core2](https://docs.m5stack.com/en/core/core2).
+Those 4 nugets provide a support for the M5 Stack's Core (gray) [M5Stack](https://docs.m5stack.com/en/products?id=core), [M5StickC](https://docs.m5stack.com/en/core/m5stickc), [M5StickCPlus](https://docs.m5stack.com/en/core/m5stickc_plus) and [M5Core2](https://docs.m5stack.com/en/core/core2).
 
-They bring support for the screens as well and require to be flashed with the proper image:
+IMPORTANT: The M5Stack NuGet Package must only be used for application targeting the M5 Stack's Core (gray); the other 3 NuGets must be used individually depending on your target according its name.
+NOTE1: Before trying to add NuGet packages to your projects and/or before flashing the devices (see next section) using MS Visual Studio (VS), open VS > Tools > Options > NuGet Package Manager > Package Sources  and make sure that it contains an entry pointing to https://api.nuget.org/v3/index.json , otherwise add it.
+NOTE2: When invoking VS > Project > Manage NuGet Packages ... make sure that the "include prerelease" checkbox is clicked/selected and that in the Package source drop-down menu (right upper corner) "nuget.org" is selected.  
```

Comment:
This is required if you're targeting preview versions, which may not be the case... Your struggle with this came from the fact that there was no stable version released when you started working on this.
```suggestion
NOTE2: When invoking VS > Project > Manage NuGet Packages make sure that in the Package source drop-down menu (right upper corner) "nuget.org" is selected. Also if you're using preview version the "include prerelease" checkbox should be clicked/selected as well.  
```